tests/records/full-record.json

```diff
@@ -0,0 +1,202 @@
+{
+  "$schema": "https://localhost/schemas/records/record-v1.0.0.json",
+  "id": "abcde-12345",
+  "conceptid": "12345-abcde",
+  "pid": {
+    "pk": 1,
+    "pid_type": "recid",
+    "obj_type": "rec",
+    "status": "R"
+  },
+  "conceptpid": {
+    "pk": 2,
+    "pid_type": "recid",
+    "obj_type": "rec",
+    "status": "R"
+  },
+  "pids": {
+    "doi": {
+      "id": "10.5281/zenodo.1234",
+      "provider": "datacite",
+      "client": "zenodo"
+    },
+    "concept-doi": {
+      "id": "10.5281/zenodo.1234",
+      "provider": "datacite",
+      "client": "zenodo"
+    },
+    "handle": {
+      "id": "9.12314",
+      "provider": "cern-handle",
+      "client": "zenodo"
+    },
+    "oai": {
+      "id": "oai:zenodo.org:12345",
+      "provider": "zenodo"
+    }
+  },
+  "metadata": {
+    "resource_type": {
+      "type": "publication",
+      "subtype": "article"
+    },
+    "creators": [{
+      "name": "Nielsen, Lars Holm",
```

Comment:
will we enforce a validation here? or fill in the other two fields from inference?

Reply:
Yes, some sort of validation + maybe automatic filling (useful to distinguish personal vs organizational)